src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,97 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import React from 'react';
+import { Person } from '../types';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug } = useParams();
+
+  const allNamesOnServer = people.map(person => person.name);
+
+  const getParentSlug = (choseName: string) => people
+    .find(person => person.name === choseName)?.slug;
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            name,
+            sex,
+            born,
+            died,
+            fatherName,
+            motherName,
+            slug: currentSlug,
+          } = person;
+
+          return (
+            <tr
+              key={currentSlug}
+              data-cy="person"
+              className={classNames({
+                'has-background-warning': currentSlug === slug,
+              })}
+            >
+              <td>
+                <Link
+                  to={`../${currentSlug}`}
+                  className={classNames({
+                    'has-text-danger': sex === 'f',
+                  })}
+                >
+                  {name}
+                </Link>
+              </td>
+
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+
+              {motherName && allNamesOnServer.includes(motherName) ? (
```

Comment:
I think if we find a name`getParentSlug(motherName) ` then its the same check as 
` allNamesOnServer.includes(motherName)`

I mean we can replace it but put `getParentSlug(motherName) ` into const in order to invoke this function one time